packages/code-du-travail-frontend/src/common/Feedback/feedback.service.js

```diff
@@ -8,7 +8,7 @@ export function postFeedback(url, data) {
       Accept: "application/json",
       "Content-Type": "application/json"
     },
-    body: JSON.stringify(data)
+    body: JSON.stringify({ ...data, hostname: window.location.hostname })
```

Comment:
I would prefer to add  `hostname` where we gather the data

Reply:
also, I thought we already have the host in data to send...

Reply:
Well now that you make me think about it, we do send an `url`param which is `pathname + query` and we put in every config data object. Maybe we should send the complete path in one param here and that's it.

Reply:
maybe we should ask @ClementChapalain if he prefers to have it splited or not... 
Yet, I think hostname should be added to data at the same place other data are added too

Reply:
IMHO, the issue is a global one here, whatever the call. We want all the forms to have the env so we can check wether they are from localhost dev etc. So it would be error prone to manually add it in data everytime

Reply:
as long a I can easily split the env afterwards in the spreadsheet (with a simple regex for example : https://support.google.com/docs/answer/3098244?hl=fr) it's ok for me :)

Reply:
So I would say go send the compte url ?

Reply:
Ok so I added the complete URL everytime we call the postFeedback. And I made sure it is only used when we know it's not in an SSR context (i.e. in user action's callback only)